src/js/components/Accordion.js

```diff
@@ -63,7 +70,12 @@ export default class Accordion extends Component {
 
 Accordion.propTypes = {
   animate: PropTypes.bool,
+  disabled: PropTypes.bool,
   openMulti: PropTypes.bool,
+  icon: PropTypes.oneOfType([
+    PropTypes.element,
+    PropTypes.bool
```

Comment:
What's the rationale for having no icon? Is it for the disabled + all expanded case?